src/SwiftUI/SwiftUI/Color.cs

```diff
@@ -0,0 +1,226 @@
+﻿using System;
+using System.Buffers;
+using System.Runtime.InteropServices;
+
+using Swift.Interop;
+
+namespace SwiftUI
+{
+	using static Color;
+
+	public enum RGBColorSpace
+	{
+		DisplayP3,
+		RGB,
+		RGBLinear,
+	};
+
+	[SwiftImport (SwiftUILib.Path)]
+	public unsafe partial class Color : View
+	{
+		public IntPtr Data { get; private set; }
+
+		#region Static Colour Spaces
+		public static IntPtr ColorSpaceDisplayP3 {
+			get {
+				var fullMeta = RGBColorSpaceMetadata (0);
+				var meta = &fullMeta->Metadata;
+				return GetColorSpaceDisplayP3 (meta);
+			}
+		}
```

Comment:
We should not expose this.. the public API should just use the `RGBColorSpace` enum defined above

Reply:
As it looks like these ColorSpaces will take a little time to workout, I've removed all the `RGBColorSpace` constructors for now, so a working version can go in. Other constructors have been tested and work.